packages/v4/src/content/community/extensions-data.js

```diff
@@ -0,0 +1,58 @@
+export const communityExtensions = [
+  {
+    "component": "Catalog View",
+    "description": "Implements a catalog view using PatternFly components",
+    "tags": ["App services"],
+    "links": [
+      {
+        "name": "Doc",
+        "href": "https://patternfly-react.surge.sh/extensions/catalog-view-item-header"
```

Comment:
Some small copy edits for the catalog view extension examples:

- "Catalog view item header": In the "With vendor description" example, we should capitalize the "F" in PatternFly.

- "Catalog view filter side panel": Two of the headings in the example should be rewritten in sentence case: _Payment options_ and _Fuel mileage_.

- "Catalog view properties side panel": Let's rewrite these headings in sentence case: _Operator version_, _Certified level_, _Health index_, _Container image_, and _Created at_.

Reply:
I can open issues to address these